src/sections/dataset/dataset-alerts/DatasetAlerts.tsx

```diff
@@ -1,34 +1,13 @@
-import { Alert } from '@iqss/dataverse-design-system'
 import { DatasetAlert } from '../../../dataset/domain/models/Dataset'
-import { useTranslation } from 'react-i18next'
-import styles from './DatasetAlerts.module.scss'
-import parse from 'html-react-parser'
+import { useDatasetAlertContext } from '../DatasetAlertContext'
+import { Alerts } from '../../alerts/Alerts'
 
 interface DatasetAlertsProps {
   alerts: DatasetAlert[]
 }
 
 export function DatasetAlerts({ alerts }: DatasetAlertsProps) {
-  const { t } = useTranslation('dataset')
-
-  return (
-    <div className={styles.container}>
-      {alerts.map((alert: DatasetAlert, index) => {
-        const translatedMsg = alert.dynamicFields
-          ? t(`alerts.${alert.message}.alertText`, alert.dynamicFields)
-          : t(`alerts.${alert.message}.alertText`)
-        const translatedHeading = t(`alerts.${alert.message}.heading`)
-        const alertKey = `alert-${index}`
-        return (
-          <Alert
-            key={alertKey}
-            variant={alert.variant}
-            customHeading={translatedHeading}
-            dismissible={false}>
-            {parse(translatedMsg)}
-          </Alert>
-        )
-      })}
-    </div>
-  )
+  const statusAlerts = useDatasetAlertContext()
+  alerts = alerts.concat(statusAlerts.datasetAlerts)
+  return <Alerts alerts={alerts}></Alerts>
```

Comment:
Ok, I see what you mean, that makes sense.  I had to add some code to make sure that the same alert isn't added multiple times, because it seems that the render of DatasetAlerts.tsx is called multiple times when the page is loaded. 
I agree about making DatasetAlert more general - I renamed the DatasetAlert type and moved it to it's own file, Alert.ts.  Do you think there is a need for subclassing Alert, and AlertMessageKey, that way for example, Dataset.alerts could still contain alerts of type DatasetAlert, and DatasetAlert would only allow a subset of messages?  I think maybe for now it's ok to keep it general, and subclass if the alerts need be be more organized, what do you think?

Reply:
The changes look good! 

I like the idea of subclassing Alert. But as you said, it's fine to keep it like this since there are no other types of alerts. We can add the subclasses once there are other type of alerts